fraud/proof.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/celestiaorg/celestia-node/header"
 )
 
+type ErrFraudExists struct {
+	Type  ProofType
```

Comment:
You can take the type out of the first proof in the array below

Reply:
What if Len(proofs) == 0 ?

Reply:
Then the error is invalid and code the code that made it is incorrect as well and should panic.

Reply:
removed Type param as you suggested.